star-wars-films/src/PersonCard/PersonCard.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PersonCard from './PersonCard';
+import ReactTestRenderer from 'react-test-renderer';
+import {
+  MemoryRouter
+} from "react-router-dom";
+import PropTypes from 'prop-types';
+
+describe('PersonCard component', () => {
+  it('should render', () => {
+    const mockData = {
+      "name": "Luke Skywalker",
+      "url": "https://swapi.co/api/people/1/"
+    };
+    const component = ReactTestRenderer.create(
+      <MemoryRouter>
+        <div>
+          <PersonCard people={mockData} />;
```

Comment:
I tried multiple times to pass in data from an actual api call here. It never worked. In the end I mocked the data. I'd be curious to see how to do this with an actual api call.